core/src/main/java/io/opencensus/stats/Measure.java

```diff
@@ -31,7 +35,10 @@
       Function<? super MeasureLong, T> p1);
 
   /**
-   * Name of measure, as a {@code String}.
+   * Name of measure, as a {@code String}. Should be a ASCII string with a length no greater than
+   * 255 characters.
+   *
+   * <p>Suggested format for name: {@code <web_host>/<path>}.
```

Comment:
add rationale for suggested format? For example, this avoids problems with dots (ex in some backends like elasticsearch, dots hint at an object structure and interfere with indexing). Not sure your rationale..